Bottom: 0851770222279ac71011d07cf5f830cc2b849493
Top:    f8df6cf62156ed2b2b668cccf939b72c759568ef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 15:21:23 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2617f89..69bd21d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -939,8 +939,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   return FALSE;
 }
 
-<<<<<<< current
-=======
 void
 ghid_pinout_preview_init (GhidPinoutPreview *preview)
 {
@@ -953,7 +951,6 @@ ghid_pinout_preview_init (GhidPinoutPreview *preview)
                                 GDK_GL_RGBA_TYPE);
 }
 
->>>>>>> patched
 gboolean
 ghid_pinout_preview_expose (GtkWidget *widget,
                             GdkEventExpose *ev)
